client/src/components/Class/ClassDashboard/ClassDashboard.tsx

```diff
@@ -0,0 +1,151 @@
+import {
+  UserOutlined,
+  BarChartOutlined,
+  ReconciliationOutlined,
+  QuestionCircleOutlined,
+  DeliveredProcedureOutlined,
+  CarryOutOutlined,
+  BellFilled,
+  MenuOutlined,
+} from "@ant-design/icons";
+import React from "react";
+import { Outlet, NavLink } from "react-router-dom";
+import type { MenuProps } from "antd";
+import { Breadcrumb, Layout, Menu } from "antd";
+import Avatar from "../../../assets/avatar.png";
+import Logo from "../../../assets/Logo.png";
+import "./ClassDashboard.css";
+
+const { Header, Content, Sider } = Layout;
+
+const styles = {
+  navLinks: {
+    width: "100%",
+    display: "flex",
+    justifyContent: "center",
+    alignItems: "center",
+  },
+  navIcons: { fontSize: "1.5rem" },
+  header: {
+    display: "flex",
+    justifyContent: "space-between",
+    alignItems: "center",
+    gap: "3rem",
+    backgroundColor: "#fff",
+  },
+  headerFirstSection: {
+    width: "15rem",
+    height: "100%",
+    display: "flex",
+    justifyContent: "space-between",
+    alignItems: "center",
+  },
+  headerSecondSection: {
+    width: "6rem",
+    height: "100%",
+    display: "flex",
+    justifyContent: "space-between",
+    alignItems: "center",
+  },
+  menuIcon: {
+    fontSize: "27px",
+    fontWeight: "bold",
+    height: "18px",
+    color: "#0CBE8A",
+  },
+  logo: { width: "10rem" },
+  bellIcon: { fontSize: "23px", color: "#767672" },
+  notificationsDot: {
+    width: "0.6rem",
+    height: "0.6rem",
+    borderRadius: "50%",
+    left: "1rem",
+    top: "1rem",
+    backgroundColor: "#0CBE8A",
+  },
+  navbar: {
+    height: "100%",
+    borderRight: 0,
+    display: "flex",
+    gap: "2rem",
+    color: "#B8C5D3",
+  },
+  layout: { padding: "0 24px 24px" },
+  breadcrumb: { margin: "16px 0" },
+  layoutContent: {
+    padding: 24,
+    margin: 0,
+    minHeight: 280,
+  },
+};
```

Comment:
to keep your code clean you should move the style to an external CSS file

Reply:
It's valid both ways, but I'll move them if you put your'all styles in a CSS file.